autogen/oai/client.py

```diff
@@ -72,7 +73,13 @@ class OpenAIWrapper:
     total_usage_summary: Optional[Dict[str, Any]] = None
     actual_usage_summary: Optional[Dict[str, Any]] = None
 
-    def __init__(self, *, config_list: Optional[List[Dict[str, Any]]] = None, **base_config: Any):
+    def __init__(
+        self,
+        *,
+        config_list: Optional[List[Dict[str, Any]]] = None,
+        streaming_transmitter: Optional[StreamingTransmitter] = None,
```

Comment:
`streaming_transmitter` goes into the `extra_kwargs` list see above. It will be taken out of the `base_config` dictionary in the constructor. See how `cache` is used in `create`